src/blog/2024/09/node-red-version-control-with-snapshots.md

```diff
@@ -0,0 +1,238 @@
+
+## Setting a Device Target Snapshot
+
+When you set a device target snapshot, all devices associated with the instance will be restarted and updated to run that specified snapshot. This gives you full control over when devices receive the latest changes from your development instance.
+
+With this feature, you can continue developing and testing your flows in FlowFuse without immediately impacting your devices. Once you're confident the flow is ready, you can push the changes to the devices by setting the target snapshot.
+
+Device target snapshots can be assigned either during the creation of a snapshot or at a later time.
+
+### To set the already creted Snapshot as a Device Target Snapshot:
+
+1. Go to the **Snapshots** tab and click on the three-dot icon next to the desired snapshot.
+
+![Image showing the 'Set as Device Target' option in the menu](./images/set-targe-snapshot-option.png){data-zoomable}
+*Image showing the 'Set as Device Target' option in the menu*
+
+2. Select **Set as Device Target** from the menu, prompt will open to conform then click on to the "Set Target".
+
+![Image showing the green status indicating how many devices have deployed this snapshot option in the menu.](./images/status-showing-the-target-snapshot.png){data-zoomable}
+*Image showing the green status indicating how many devices have deployed this snapshot option in the menu.*
+
+Once done you will be able to see the green mark in that snapshot showing on how much devices it is deployed on.
+
+## Common Mistakes to Avoid
+
+1. **Neglecting Regular Backups**: Don’t skip those snapshots! Regularly create backups, especially before making significant changes, moving devices through instances or applications, or disabling developer mode. Think of it as your safety net—ensuring you can always bounce back to a stable state if unexpected issues pop up.
+
+2. **Overlooking Auto-Snapshot Limits**: Did you know that your **auto snapshots** have limits? Be mindful of how many you can retain, as older **auto snapshots** will be automatically deleted. If you have important **auto snapshots**, save them locally to keep them safe. Remember, manually created snapshots have no limits, so take advantage of that!
+
+3. **Inadequate Secret Key Security**: Your secret key is your golden ticket! When downloading snapshots locally, **securely store that key**. Losing it could mean losing access to your snapshot and your ability to recover your work. Treat it like a password—keep it safe!
+
+4. **Taking Device Snapshots in the Instance Snapshots Tab**: Make sure you're in the right place! Always take device snapshots in the **Developer Mode** tab. Some users mistakenly capture snapshots for devices in the instance associated with the devices, which only captures the snapshot for the instance itself. This can lead to losing valuable work done on specific device. 
```

Comment:
Slightly misleading. As before, the instance device, when in fleet mode, should be running the snapshot set as "target" (will actually be running "active snapshot").

It is worth noting that if the device is in developer mode it MAY have modified flows and therefore using the correct snapshot action for an instance device becomes important.